modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -36,6 +37,9 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   async updateState({props, oldProps}) {
+    if ('onTileLoadFail' in props) {
+      log.warn('`onTileLoadFail` is deprecated. use `onTileError` instead.');
```

Comment:
- Move to `initializeState` - we usually only check deprecated props once
- Use `log.removed`
- `log.warn` returns a function. You need to call it e.g. `log.warn(...)()`.